packages/gp2-components/src/molecules/__tests__/ExpandableText.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import ExpandableText from '../ExpandableText';
+
+describe('ExpandableText', () => {
+  const text = 'this is a text';
+  it('renders the children', () => {
+    render(<ExpandableText>{text}</ExpandableText>);
+    expect(screen.getByText(text)).toBeVisible();
+  });
+  it('renders show more if text height is larger than max height', () => {
+    const ref = { current: { scrollHeight: 125 } };
+
+    Object.defineProperty(ref, 'current', {
+      set(_current) {
+        this.mockedCurrent = _current;
+      },
+      get() {
+        return { scrollHeight: 125 };
+      },
+    });
+    jest.spyOn(React, 'useRef').mockReturnValue(ref);
```

Comment:
Should be test this using Playwright?

Reply:
this could have been tested using playwright or the browser tests, but I believe the current test does cover the use case

Reply:
okay, let's go with this and then check with @kmaid when he is back.